packages/big-design/tests/utils.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, RenderOptions } from '@testing-library/react';
+import React from 'react';
+import { UIDReset } from 'react-uid';
+
+const BigDesignWrapper: React.FC = ({ children }) => {
+  return <UIDReset>{children}</UIDReset>;
+};
+
+const customRender = (ui: React.ReactElement<any>, options: RenderOptions = {}) =>
+  render(ui, { wrapper: BigDesignWrapper, ...options });
+
+// re-export everything
+export * from '@testing-library/react';
```

Comment:
cool trick (🍹 although not sure the comment above adds anything, the code itself is pretty clear)